src/System.Net.Security/tests/FunctionalTests/ClientAsyncAuthenticateTest.cs

```diff
@@ -24,6 +24,17 @@ public ClientAsyncAuthenticateTest()
             _log = TestLogging.GetInstance();
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "CI uses old Framework version, which doesn't support using SslProtocols.None for SystemDefaultTlsVersions behavior")]
+        public async Task ClientAsyncAuthenticate_SslStreamClientServerNone_UseStrongCryptoSet()
+        {
+            SslProtocols protocol = SslProtocols.None;
+            await ClientAsyncSslHelper(protocol, protocol);
+
+            // Additional manual verification.
+            // Step into the code and verify that the 'SCH_USE_STRONG_CRYPTO' flag is being set.
```

Comment:
Is there any way we could automate this?  For example, could we try to connect to a loopback server using SslStream configured to use the weaker crypto and verify that the connection fails when using SslProtocols.None? I don't know if we expose the necessary knobs on SslStream to enable doing that.

Reply:
SslStream doesn't expose any public APIs for configuring the crypto. From SCHANNEL experts:

>  the current "SystemDefault' will actually use strong crypto settings on current Windows OS's.

But they advised us that we should always pass in the SCH_USE_STRONG_CRYPTO flag when using 'SystemDefault'. There is no behavior difference that can be captured by test for this change.

We could consider adding the scenario you mentioned (SystemDefault with weak crypto) to [SslStreamSystemDefaultsTest.cs](https://github.com/dotnet/corefx/blob/master/src/System.Net.Security/tests/FunctionalTests/SslStreamSystemDefaultsTest.cs) in a different PR, more like verifying SystemDefaultTlsVersions behavior. (if feasible)